backend/services/src/shared/util/configurationSettings.service.ts

```diff
@@ -0,0 +1,70 @@
+import { HttpException, HttpStatus, Injectable, Logger } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { Repository } from "typeorm";
+import { BasicResponseDto } from "../dto/basic.response.dto";
+import { ConfigurationSettings } from "../entities/configuration.settings";
+import { ConfigurationSettingsType } from "../enum/configuration.settings.type.enum";
+import { HelperService } from "./helpers.service";
+
+@Injectable()
+export class ConfigurationSettingsService {
+  constructor(
+    @InjectRepository(ConfigurationSettings)
+    private configSettingsRepo: Repository<ConfigurationSettings>,
+    private logger: Logger,
+    private helperService: HelperService
+  ) {}
+
+  async getSetting(type: number) {
```

Comment:
If you add optional parameter to parse default value and if the value is null return the default value will simplify default behaviour handle

Reply:
fixed